http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -163,12 +163,14 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       // Retrieve parent workspaceId for the google project
       parentWorkspaceId <- samService.lookupWorkspaceParentForGoogleProject(userInfo.accessToken.token, googleProject)
 
+      leoToken <- authProvider.getLeoAuthToken
+      leoEmail <- samService.getUserEmail(leoToken)
```

Comment:
It would be good to leave a comment here explaining what the leoEmail will be used for. I am assuming that the answer is in the sam client, but would be good to isolate the GCP from the Azure case, even in a small comment